source/faq.rst

```diff
@@ -19,6 +19,18 @@ Authentication via CLI or API requires a valid Check-in token. The
 FedCloud Check-in client allows you to get one as needed. Check the
 :doc:`auth` guide for more information.
 
+Is OCCI still supported?
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+OCCI is now deprecated as API for the EGI Cloud providers using OpenStack. If
+you still rely on OCCI for your access, please contact us for support on the 
```

Comment:
maybe would be useful to have a contact email here directly

Reply:
done